app/src/main/kotlin/at/bitfire/davdroid/ui/AccountsModel.kt

```diff
@@ -92,7 +96,33 @@ class AccountsModel @AssistedInject constructor(
     private val workManager = WorkManager.getInstance(context)
     private val runningWorkers = workManager.getWorkInfosFlow(WorkQuery.fromStates(WorkInfo.State.ENQUEUED, WorkInfo.State.RUNNING))
 
-    val accountInfos: Flow<List<AccountInfo>> = combine(accounts, runningWorkers) { accounts, workInfos ->
+    @OptIn(ExperimentalCoroutinesApi::class)
+    private val accountsPendingInSyncFramework: Flow<List<Account>> =
+        accounts.flatMapLatest { accounts ->
+            if (accounts.isEmpty())
+                flowOf(emptyList())
+            else {
+                // For each account, create a Flow<Boolean> emitting the state on whether it has a pending sync.
+                val pendingSyncCheckFlows: List<Flow<Account?>> = accounts.map { account ->
```

Comment:
Hm maybe `pendingSyncAccountsFlows` as a name for better understanding? But doesn't change that much. The "check" is somehow irritating

Reply:
I have renamed it and added more comments.